### ISSUE TITLE
Position of images when zoomed

### ISSUE THREAD

**Issue by adamelnaggar** [tholman/intense-images#69]
Hi @tholman thanks very much for developing Intense Images it's really great and fits my needs almost perfectly.

An issue I'm having though is that on clicking the image to zoom it loads the image at the vertical centre of the image (on a vertical image). 

I'm using the library to zoom into screenshots of entire websites that I have placed on a page (on desktop only) and want the image to load at the top on click, so that the user can scroll down from the top through the image. 

So basically I'm trying to set the initial position of a vertical image to be the top of the image when opened full screen..

I've tried playing around with the code but can't figure out how to accomplish this, although it seems like it could be an easy fix. Is this a feature you would consider adding? Or is there an easy fix I'm missing?

Cheers! 🍻

**Comment by adamelnaggar** [comment]
The issue I'm having is more clear on [this page](https://adamwrites.co/north-sea-case-study).
